src/shared/api/http.ts

```diff
@@ -55,6 +55,10 @@ export function fetchBlocks(params?: PaginationParams): Promise<Paginated<BlockS
   return get('/blocks', params);
 }
 
+export function fetchBlocksDetails(block_hash: string): Promise<Block> {
```

Comment:
Look at the [API reference](https://github.com/soramitsu/iroha2-block-explorer-backend/blob/57da68535e2043e0694ae4049ffac83bae2565d1/api.md#get-apiv1blocksheight-or-hash):

> `/api/v1/blocks/{height or hash}`

Therefore, to make `fetchBlockDetails` more accurate, it should be:

```suggestion
export function fetchBlocksDetails(height_or_hash: string | number): Promise<Block> {
```

Reply:
acknowledged and committed 👍🏼